src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -309,8 +301,7 @@ limitations under the License.
       <arg name="arg2" type="SimpleEnum"/>
     </command>
 
-    <command source="client" code="0x0F" name="TestNullableOptionalRequest"
-             optional="false" response="TestNullableOptionalResponse">
+    <command source="client" code="0x0F" name="TestNullableOptionalRequest" optional="false" response="TestNullableOptionalResponse">
```

Comment:
Why the random reformatting?  Looks like your editor is automatically reformatting stuff, but it was automatically reformatted by other people's editors already, so we just end up with an edit war.  Please remove the changes that are not related to what is actually being changed here.